### PR TITLE
active experience refactoring

### PR DESCRIPTION
**Pull request by FernandoArevaloE**
* changes made to allow getting an experience if active. Doing this it will possible handle multiple experiences with an started game. Covers case "Determine if an experience can be played"
* removed code for jackpots from team resources since that logic was added under inventory resources
* updated missing documentation to show correct urls for testing scenarios
* added missing documentation for "Exchange parts by drawing tickets"
* removed join with table player ni getExperience query because of constraints and primary key

### CODE REVIEW

File: app/src/service/experience.js [hunk 1]
```diff
@@ -34,13 +34,24 @@ const getExperienceById = async (experienceId) => {
...
 
-    const experience = await experienceDAO.getExperienceByPlayerIdAndActivationFlag(playerId, true);
-    if (!experience) throw new NotFound();
+    const experiences = await experienceDAO.getExperienceByPlayerIdAndActivationFlag(playerId, true);
+    if (!experiences) throw new NotFound();
 
-    return experience;
+    if (!experienceId) {
+        return {
+            data: experiences
```
Comment (andymotagl) [inline app/src/service/experience.js:45]: This would be mad the consumer. Because when there is no _filter_ the result looks different when it is

Comment (FernandoArevaloE) [reply app/src/service/experience.js:45]: what if we have two methods? one to get all active experience and other to get  the specific one

all -> `/:playerId/experience/all`

specific -> `/:playerId/experience?id=...`

what do you think @andymotagl & @henrymcarvajal ?

Comment (andymotagl) [reply app/src/service/experience.js:45]: Would be better. And the one for single can the experienceId be in the path

Comment (FernandoArevaloE) [reply app/src/service/experience.js:45]: done. Becuase of conflict I created urls

**single one active:** 

`http://localhost:3000/v1/player/{player}/experience/{experience}`

**all active:** 

`http://localhost:3000/v1/player/{player}/experience`

**Review by andymotagl — APPROVED** [review]

**Pull request merged**